userprofile/tests/test_ui.py

```diff
@@ -22,3 +24,27 @@ def test_update_button(self):
         url = reverse('userprofile:detail')
         expected = [(self.local_user, 1), (self.ldap_user, 0)]
         self.assert_element_count(expected, url, 'sodar-user-update-btn')
+
+
+class TestUserSettings(TestUIBase):
+    """Tests for user settings page"""
+
+    def setUp(self):
+        super().setUp()
+        # Create users
+        self.local_user = self.make_user('local_user', False)
+        self.ldap_user = self.make_user('user@EXAMPLE', False)
+
+    def test_settings_label_icon(self):
+        """Test existence of settings label icon"""
+        url = reverse('userprofile:settings_update')
+        self.login_and_redirect(
+            self.superuser, url, wait_elem=None, wait_loc='ID'
+        )
+        WebDriverWait(self.selenium, 15)
+
+        logo = self.selenium.find_element(
```

Comment:
I'm nitpicking here, but technically we're looking for an "icon" here, not a "logo". Might be confusing for someone reading/updating this test case later on :)